hazelcast-spring/src/main/resources/hazelcast-spring-4.0.xsd

```diff
@@ -3285,6 +3295,49 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="bitsTimestamp" default="41" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    Sets the timestamp component bit length, default 41 bits.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:simpleType>
+                <xs:restriction base="xs:int">
+                    <xs:minInclusive value="1" />
+                </xs:restriction>
+            </xs:simpleType>
+        </xs:attribute>
+        <xs:attribute name="bitsSequence" default="6" use="optional">
+            <xs:annotation>
+                <xs:documentation>
+                    Sets the sequence component bit length, default 6 bits.
+                </xs:documentation>
+            </xs:annotation>
+            <xs:simpleType>
+                <xs:restriction base="xs:int">
+                    <xs:minInclusive value="1" />
```

Comment:
we could also use 0 sequence bits if the user is fine with 1 ID per millisecond, I think it's perfectly useful.

Reply:
Yes, 0 works well here.